src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlBlock.kt

```diff
@@ -132,9 +136,11 @@ class RitualBowlBlock : Block(
                 player.heldItemMainhand.damageItem(1, player, { it.sendBreakAnimation(handIn) })
             }
 
+            val recipe = worldIn.server!!.recipeManager.getRecipe<IInventory, BowlRitualRecipe>(ModRecipes.BOWL_RITUAL_TYPE!!, tile, worldIn)
+
             if (!inProgress) {
                 worldIn.setBlockState(pos, state.with(BlockStateProperties.ENABLED, true))
-                worldIn.pendingBlockTicks.scheduleTick(pos, ModBlocks.RITUAL_BOWL!!, 200)
+                worldIn.pendingBlockTicks.scheduleTick(pos, ModBlocks.RITUAL_BOWL!!, recipe.map { it.duration }.orElse(BowlRitualRecipe.DEFAULT_DURATION))
```

Comment:
Maybe add more line breaks to increase readability

Reply:
Formatting is enforced by ktlint. If there are any additional rules, we should extend the ruleset.

Reply:
Added #21 to keep an eye on this.